pkg/restapi/vc/operation/persistence.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package operation
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/trustbloc/edge-core/pkg/storage"
+)
+
+const (
+	keyPattern       = "%s_%s"
+	profileKeyPrefix = "profile"
+)
+
+// NewCredentialRecorder returns new credential recorder instance
+func NewCredentialRecorder(store storage.Store) *CredentialRecorder {
+	return &CredentialRecorder{store: store}
+}
+
+// CredentialRecorder takes care of features to be persisted for credentials
+type CredentialRecorder struct {
+	store storage.Store
+}
+
+// SaveProfile saves issuer profile to underlying store
+//
+// Args:
+//
+//
+// Returns:
+//
+// error: error
```

Comment:
don't need this part of the comment. Or perhaps expand on it so it provides some more useful info if you think it's needed

Reply:
done